man/systemd.network.xml

```diff
@@ -872,6 +872,17 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv6RetransTime=</varname></term>
```

Comment:
we generally avoid unnecssary appreviations in our settings names, unless the are very well established. 

Also, we for time value we generally want to use our high level parsers (which can parse strings such as "5min 30s"), and if the strings it takes are specified without unit, then we want this to be understood as seconds as the default time unit, and that again should be indicated in the setting name, i.e. with a suffix of "Sec". 

Hence, please rename this to `IPv6RetransmissionTimeSec=`.

Reply:
Done.

Reply:
Curiously, do the high-level parsers already support units of "ms" or does the user have to specifiy a decimal value, like 0.1 (for 100ms)? While I can experiment with that, I'd rather know the official position on this.